storage/libradb/src/lib.rs

```diff
@@ -134,6 +134,8 @@ impl LibraDB {
 
         let path = db_root_path.as_ref().join("libradb");
         let instant = Instant::now();
+        // TODO: TODO: either return a Result or let the caller open the DB and deal with the failure.
+        // Panic is not an option here.
```

Comment:
@zamsden  again?

Reply:
My guess is that we only do this at initialization, so it might not be a big deal if we fail here.